server/services/team/service.ts

```diff
@@ -584,12 +583,16 @@ export class TeamService extends ServiceClass implements ITeamService {
 		await this.TeamMembersModel.deleteByUserIdAndTeamId(userId, teamId);
 	}
 
-	async removeMembers(teamId: string, members: Array<ITeamMemberParams>): Promise<boolean> {
+	async removeMembers(uid: string, teamId: string, members: Array<ITeamMemberParams>): Promise<boolean> {
 		const team = await this.TeamModel.findOneById(teamId, { projection: { _id: 1, roomId: 1 } });
 		if (!team) {
 			throw new Error('team-does-not-exist');
 		}
 
+		const membersIds = members.map((m) => m.userId);
+		const usersToRemove = await this.Users.find({ _id: { $in: membersIds } }, { projection: { _id: 1, username: 1 } }).toArray();
```

Comment:
the new changes are looking good, it is definitely better finding all members at once :clap: 

we just need to avoid using custom queries, so in this case I'd recommend using `this.Users.findByIds`.. looks like it doesn't exists, so you'd to create it.. :wink: